### PR TITLE
Added Game::MissionFlag property and renamed Player::IsOnMission

### PR DESCRIPTION
**Pull request by kagikn**

### CODE REVIEW

File: source/Player.cpp [hunk 1]
```diff
@@ -59,9 +59,9 @@ namespace GTA
 	{
 		return Native::Function::Call<bool>(Native::Hash::IS_PLAYER_FREE_AIMING, this->Handle);
 	}
-	bool Player::IsOnMission::get()
+	bool Player::CanStartMission::get()
```
Comment (JohnnyCrazy) [inline source/Player.cpp]: Wait, so was this inverted the whole time? true in `CanStartMission` is not the same as true in `IsOnMission`

Comment (kagikn) [inline source/Player.cpp]: Yeah, it was inverted **the whole time.**

Comment (JohnnyCrazy) [inline source/Player.cpp]: Just saw the exclamation mark, wasn't inverted, my bad.

So any reason to actually change the name? Is there any case where `CAN_PLAYER_START_MISSION` returns false with not being in a mission?

**Comment by kagikn** [comment]
it's because CAN_PLAYER_START_MISSION not always return false with not being in a mission(e.g. When the player is ragdoll). whereas GET_MISSION_FLAG always returns true with being in a mission.
when I made a script to supply the player with armor and health when the player start a mission, I noticed that.

**Comment by JohnnyCrazy** [comment]
Alright! All should be good then :+1:

**Pull request merged**